plutus-benchmark/bls12-381-costs/test/bls12-381-costs.golden

```diff
@@ -103,7 +103,7 @@ VRF example
 
     n     Script size             CPU usage               Memory usage
   ----------------------------------------------------------------------
-    -     795   (4.9%)      1548749705  (15.5%)          405291   (2.9%) 
+    -     796   (4.9%)      1548887705  (15.5%)          405891   (2.9%) 
```

Comment:
@kwxm for some reason the UPLC inliner change changes the bls and ed25519 costs? Not by much though.